### DISCUSSION TITLE
Will there be plans for ng-zorro to extract basic components similar to react-component in the future?

### DISCUSSION THREAD

**Discussion by stygian-desolator** [NG-ZORRO/ng-zorro-antd#6087] [General] [answered]
Will there be plans for `ng-zorro` to extract basic components similar to `react-component` in the future?

**Comment by vthinkxie** [comment, marked as answer]
ng-zorro-antd works under mono repo now, since all components can import independently in ng-zorro-antd, it won't bring too much benefit to extract basic components like react-component IMO

feel free to comment if you have any idea or suggestion
